File: Methods/Methods-Python/skew.py
import math


def skew(data):

    suma = 0
    sumPD = 0
    sumSq = 0

    for i in range(0, len(data)):
        suma += data[i]
        sumSq += data[i] * data[i]

    mean = suma / len(data)
    standardDeviation = math.sqrt((sumSq - mean * suma) / len(data))

    for i in range(0, len(data)):
        sumPD += math.pow(data[i] - mean, 3)

    moment3 = sumPD / len(data)

    return moment3 / (standardDeviation * standardDeviation * standardDeviation)

File: Methods/Methods-Java/Java source-code/skew.java


public class skew {
    public static double skew_m(Double[] data) {
        double sum = 0;
        double sumPD = 0;
        double sumSq = 0;
        for (int i = 0; i < data.length; i++) {
            sum += data[i];
            sumSq += data[i] * data[i];
        }
        double mean = sum / data.length;
        double standardDeviation = Math.sqrt((sumSq - mean * sum) / data.length);
        for (int i = 0; i < data.length; i++) {
            sumPD += Math.pow(data[i] - mean, 3);
        }
        double moment3 = sumPD / data.length;
        return moment3 / (standardDeviation * standardDeviation * standardDeviation);
    }
}